features/financial_assistance/step_definitions/health_coverage_steps.rb

```diff
@@ -84,6 +84,7 @@
 end
 
 Then(/^the medicare have glossary content$/) do
+  #bug logged
```

Comment:
do we need this note here? If we want to keep it, potentially add details as to what is breaking

Reply:
same goes for list of bug logged comments

Reply:
@ATBull81 yes I could removed them if you would like, it was a way for me to track if I had looked into this particular step. Would you prefer for me to remove or add additional details on the bug?

Reply:
I think I lean more details. If there is an issue constantly occurring, it may save others time from digging into it if they see the problem noted

Reply:
2nd, more details please